django/core/checks/urls.py

```diff
@@ -19,22 +19,13 @@ def check_resolver(resolver):
     """
     Recursively check the resolver.
     """
-    from django.urls import RegexURLPattern, RegexURLResolver
     warnings = []
     for pattern in resolver.url_patterns:
-        if isinstance(pattern, RegexURLResolver):
-            warnings.extend(check_include_trailing_dollar(pattern))
-            # Check resolver recursively
-            warnings.extend(check_resolver(pattern))
-        elif isinstance(pattern, RegexURLPattern):
-            warnings.extend(check_pattern_name(pattern))
-        else:
-            # This is not a url() instance
+        check_method = getattr(pattern, "check", None)
```

Comment:
You're still iterating over `resolver.url_patterns`. Another URL resolver might not have an `url_patterns` attribute.